source/log.cpp

```diff
@@ -26,6 +27,16 @@ void initLogFile()
         logFile.open(logFileName, fstream::out);
 
     const auto now = time(nullptr);
+    logFile << "Cave Story Recoded - Date of compilation : " << __DATE__ << ". Time of compilation : " << __TIME__ << '\n';
+    SDL_version compiled;
+    SDL_version linked;
+    SDL_VERSION(&compiled);
+    SDL_GetVersion(&linked);
+    logFile << "Compiled against SDL2 version " 
+    << (int)compiled.major << '.' << (int)compiled.minor << '.' << (int)compiled.patch
+    << " and linked against version " 
+    << (int)linked.major << '.' << (int)linked.minor << '.' << (int)linked.major << '\n';
```

Comment:
```suggestion
    << (int)linked.major << '.' << (int)linked.minor << '.' << (int)linked.patch << '\n';
```
With the power of GitHub Beta, I fixed your typo. Your welcome.